src/ws-dropdown/dropdown-menu.js

```diff
@@ -1,8 +1,10 @@
 import {React, Component, PropTypes} from '../imports';
 import {DropdownMenuItem} from './dropdown-menu-item';
 
-const ANIMATION_START_EVENTS = ['oAnimationStart', 'MSAnimationStart', 'animationstart'];
-const ANIMATION_END_EVENTS = ['oAnimationEnd', 'MSAnimationEnd', 'animationend'];
+const ANIMATION_START_EVENTS = ['oAnimationStart', 'MSAnimationStart',
```

Comment:
This is not the way we wrap and it doesn't seem to be a to long line. Pls configure your IDE for 120 characters